src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml

```diff
@@ -9,48 +9,91 @@
              d:DesignHeight="300"
              d:DesignWidth="400"
              mc:Ignorable="d">
+    
+    <Grid VerticalAlignment="Top">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="*"/>
+            <RowDefinition Height="Auto"/>
+        </Grid.RowDefinitions>
 
         <ListView x:Name="EquationInputList"
-              VerticalContentAlignment="Top"
-              IsItemClickEnabled="False"
-              ItemsSource="{x:Bind Equations}"
-              SelectionMode="None">
+                  IsItemClickEnabled="False"
+                  ItemsSource="{x:Bind Equations}"
+                  SelectionMode="None">
+
+            <ListView.Style>
```

Comment:
Can you add a comment what this is doing? It looks like it is just empty and doing nothing.

Reply:
```suggestion
<!-- Removes animations from the ListView Style. -->
            <ListView.Style>
```